open-grant-proposals/open-proposal-musicstudio.md

```diff
@@ -0,0 +1,90 @@
+
+# Open Grant Proposal: `Music Studio`
+
+**Name of Project: Music Studio**
+
+**Proposal Category:** `app-dev`
+
+**Proposer:** `@immartian`
+
+**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT and APACHE2 licenses?:** "Yes"
+
+# Project Description
+
+Music Studio is based on the established Musicoin Project to enable global indie musicians to manage and release their musical works and offer distributed content sharing and delivering to music listeners like Airbnb for music.
+
+Today's music industry has failed both musicians and listeners for its single pay-wall model requiring music consumers to subscribe monthly but leaving 90% of Internet users can't share the excellent works excluded by major music labels. Eventually the monopoly of the big industry players siphon over 88% of profit in the industry, leaving only 12% of profit to  musicians, however, only 2% of musicians get really well paid(top commercial ones).
+
+98% of 8 million musicians need a solution to manage their works independently and earn their revenue without middlemen; 2 billion music listeners need a solution to remove the pay-wall to enjoy the best creations all around the world without overpaying to digital music tycoons.  Musicoin Project is working on such a solution to make it possible.
+
+## Value
+
+Previously we used IPFS but didn't fully utilize its potential until Filecoin offers a grander vision that makes it more possible.
```

Comment:
Do you still use IPFS? Can you say more about not utilizing IPFS' full potential? IPFS sounds like a good fit for your use case!

Reply:
We are still using IPFS as a backend layer(offline mode, a bit strange reason from legacy), but the MUSIC Studio wants to expand to more distributed scenarios to enable musicians to own two wallets in one app. It's quite experimental, but also potential, since we heard from the community (with hundreds of musicians) that it's their interest to explore the frontier.